src/Browser/Navigators/Navigator/Assistant.js

```diff
@@ -252,106 +228,74 @@ export const suggest =
 
 export const swapSelectorAndSuggest =
   (state:Model, selector:Selector.Model):[Model, Effects<Message>] =>
-  suggestSelected
-  ( new Model
-    ( state.isOpen
-    , state.isExpanded
-    , state.query
-    , state.suggestions
-    , selector
+  suggestSelected(new Model(state.isOpen,
+     state.isExpanded,
+     state.query,
+     state.suggestions,
+     selector
     )
   )
 
-
-
 const suggestSelected =
   (state:Model):[Model, Effects<Message>] =>
-  [ state
-  , fxForSuggestion(state.query, getSelectedSuggestion(state))
+  [ state,
+   fxForSuggestion(state.query, getSelectedSuggestion(state))
   ]
 
 const fxForSuggestion =
   (query:string, suggestion:?Suggestion.Model):Effects<Message> =>
-  ( suggestion == null
+  (suggestion == null
   ? Effects.none
   : Effects.receive(Suggest(Suggestion.completion(query, suggestion)))
   )
 
 export const Suggest =
   (suggestion:Suggestion.Completion):Message =>
-  ( { type: "Suggest"
-    , suggest: suggestion
+  ({ type: 'Suggest',
+     suggest: suggestion
     }
   )
 
-
-
-export const updateSuggestion =
```

Comment:
Can you please keep this function, I know it's not used but it's going to be soon. And in any case I'd rather prune obsolete code in separate change.